packages/vsce/src/init.ts

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright 2022 Octorelease Contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as fs from "fs";
+import { IContext } from "@octorelease/core";
+import { IPluginConfig } from "./config";
+import * as utils from "./utils";
+
+export default async function (context: IContext, config: IPluginConfig): Promise<void> {
+    let packageJson;
+    try {
+        packageJson = JSON.parse(fs.readFileSync("package.json", "utf-8"));
+        context.logger.info(`VS Code extension: ${packageJson.publisher}.${packageJson.name}`);
+    } catch {
+        throw new Error(`Missing or invalid package.json in branch ${context.branch.name}`);
```

Comment:
dumb Q: will we ever get to this error?
I'm wondering if core/init already has this case covered?

Reply:
Core is language agnostic, so the only other checks for missing or invalid package.json should be in the lerna and npm plugins